README.md

```diff
@@ -1,5 +1,6 @@
 PublicLab.org
 ======
+I forked and cloned this repo.
```

Comment:
@rajvidg Can you remove this file from the PR? Thanks!!

Reply:
> @rajvidg Can you remove this file from the PR? Thanks!!

Yes I am trying to, watching some tutorials. :'(

Reply:
> @rajvidg Can you remove this file from the PR? Thanks!!

I removed the changes and reverted the file to its original one, but it got created as another commit, but the files changes section contains the required files, I tried doing it using git commands but the README file was not showing up in the branch where I committed rest of the changes, can you please look into it. @TildaDares

Reply:
@rajvidg You've resolved the issue 🎉

Reply:
> @rajvidg You've resolved the issue 🎉

Oh! thank you @TildaDares :))